falcon/asgi/response.py

```diff
@@ -14,7 +14,7 @@
 
 """ASGI Response class."""
 
-from asyncio.coroutines import CoroWrapper
+from asyncio.coroutines import CoroWrapper  # type: ignore
```

Comment:
An alternative fix is to do `from asyncio.coroutines import CoroWrapper as CoroWrapper`, but that also looks awkward.
Thoughts?

Reply:
I think it's OK as-is since it makes it clear this is a mypy workaround.